.github/workflows/ci.yaml

```diff
@@ -51,6 +51,26 @@ jobs:
         run: ./gradlew build --stacktrace
         working-directory: examples/distro
 
+  smoke-test-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11 for running Gradle
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Restore cache
+        uses: burrunan/gradle-cache-action@v1.10
+        with:
+          job-id: smokeTests
+
+      - name: Test
+        run: ./gradlew :smoke-tests:test
```

Comment:
I had one random thought that it's very hard to verify this actually tests windows - I don't know if GitHub actions supports it now but don't see any reason it couldn't run linux containers on Windows runner similar to how they run on mac runner. Is there a way to make sure our containers were set up correctly? If we log the container base os during the tests it's enough, just confirming.

Reply:
I added an assertion into tests to verify OS

Reply:
Cool!